app/ConfigRepository.php

```diff
@@ -155,16 +168,9 @@ public function getDeviceSetting($device, $key, $global_prefix = null, $default
     public function getOsSetting($os, $key, $default = null): mixed
     {
         if ($os) {
-            \LibreNMS\Util\OS::loadDefinition($os);
-
-            if (isset($this->config['os'][$os][$key])) {
-                return $this->config['os'][$os][$key];
-            }
-
             $os_key = "os.$os.$key";
```

Comment:
This may have a bug in it if $key has a . in it.  That probably shouldn't happen, but it would behave different in that case.

Reply:
or maybe the old code had a bug, and this is correct :D

Reply:
I assume this would mean a key in an OS YAML file with a "." in the key name.  I can't see any examples of these in the yaml files.